src/parser/shared/metrics/apl/conditions/not.tsx

```diff
@@ -2,11 +2,15 @@ import React from 'react';
 
 import type { Condition } from '../index';
 
-export default function not(cnd: Condition<any>): Condition<any> {
+export default function not(cnd: Condition<any>, showDont: boolean = true): Condition<any> {
   return {
     ...cnd,
     key: `not-${cnd.key}`,
     validate: (state, event, spell) => !cnd.validate(state, event, spell),
-    describe: (tense) => <>don't {cnd.describe(tense)}</>,
+    describe: (tense) => (
+      <>
+        {showDont && "don't"} {cnd.describe(tense)}
```

Comment:
per our discussion in discord i'm probably gonna mess around with alternate ways of doing this, but we can merge this PR as-is and I can refactor later if i find something i really like.